modular_skyrat/modules/cyborg/code/robot_upgrades.dm

```diff
@@ -244,3 +244,33 @@
 		var/obj/item/inducer/cyborg/inducer = locate() in target_robot.model
 		if (inducer)
 			target_robot.model.remove_module(inducer, TRUE)
+
+// Rapid Light Dispensor
+
+/obj/item/borg/upgrade/rapidlightplacer
+	name = "engineering cyborg RLD upgrade"
+	desc = "An inducer device for the engineering cyborg."
+	icon_state = "cyborg_upgrade3"
+	require_model = TRUE
+	model_type = list(/obj/item/robot_model/engineering, /obj/item/robot_model/saboteur)
+	model_flags = BORG_MODEL_ENGINEERING
+
+/obj/item/borg/upgrade/rapidlightplacer/action(mob/living/silicon/robot/target_robot, user = usr)
+	. = ..()
+	if(.)
```

Comment:
early return

Reply:
Not sure what you mean by this, this is the same code as the older RCD Modules. And I simply am unsure what you are specifying.

Reply:
He means instead of:
```
if(x)
	y(z)
```
do
```
if(!x)
	return
y(z)
```